repository/src/main/resources/db/changelog/db.changelog-rdm-2999.xml

```diff
@@ -0,0 +1,19 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-2999" author="rafal.kalita@hmcts.net">
+
+        <addColumn tableName="display_group">
+            <column name="webhook_mid_event_id" type="integer" >
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
```

Comment:
Is `webhook_mid_event_id` meant to be unique? If so, might be an idea to add a "unique" constraint...

Reply:
Following the way other webhooks are set up. They are not unique.